build.sbt

```diff
@@ -26,7 +26,13 @@ ThisBuild / githubWorkflowPublishTargetBranches := Seq(
 )
 
 ThisBuild / githubWorkflowBuild := Seq(
-  WorkflowStep.Sbt(List("fmtCheck", "test", "mimaReportBinaryIssues"))
+  WorkflowStep.Run(
+    List(
+      """sbt ++${{ matrix.scala }} fmtCheck \
```

Comment:
I guess this is the difference, note the fmtCheck doesn't have quotes around it... But does it need the `++` at all? The format check does not depend on the scala version does it?

Reply:
> The format check does not depend on the scala version does it?

selecting the scala version on format will allow us to check the format on scala version specific code i.e under`scala-2.1x` folders. Currently we don't have any so we don't really need it. I thought of separating the "checks" into a separate task and run it for one version only but ended up keeping the previous behaviour; can be handy? why not?

The quotes are an interesting case and one of the weirdest behaviours in SBT. When running sbt like this(non interactive) Scala version is not correctly selected on projects when running tasks ... quoting them helps. However, the behavior is not the same on cmds like `fmtCheck` and leads to error when it expands.

Note: from sbt console you don't need the quotes.

Reply:
Thanks for the detailed reply.